lizmap/plugin.py

```diff
@@ -521,6 +524,14 @@ def __init__(self, iface):
                 'manager': None,
             }
         }
+
+        # Set some tooltips
+        tooltip = tr(
+            'By default the layer is visible for all groups in Lizmap. '
+            'If a comma separated list of groups is defined, the layer will be visible only for these groups.')
```

Comment:
Can you add the following precision in the tooltip: `Use Lizmap Web Client group ids and not labels.`

Reply:
Indeed ! Thanks !